ksql-engine/src/main/java/io/confluent/ksql/QueryEngine.java

```diff
@@ -56,7 +56,6 @@
 import io.confluent.ksql.planner.LogicalPlanner;
 import io.confluent.ksql.planner.plan.KsqlStructuredDataOutputNode;
 import io.confluent.ksql.planner.plan.PlanNode;
-import io.confluent.ksql.util.KsqlConfig;
```

Comment:
@dhruvilshah3 removed the other instance of this in his PR. We should only merge one of these.

Reply:
nvm - just saw your comment on his PR